less/transferwise.less

```diff
@@ -36,6 +36,27 @@ address {
 }
 .footer {
     background-color: @gray-base;
+
+	.icon {
+		font-size: 50px !important;
+		margin: 0 5px 0 5px;
+	}
+
+	h5 {
+		a {
+			color: inherit !important;
+			font-weight: inherit;
+		}
+	}
+
+	a {
+		text-decoration: none;
+		font-weight: normal;
```

Comment:
We actually have some styles for 'secondary' links `a.secondary`, which we could reuse for any links inside `.footer`?

![image](https://cloud.githubusercontent.com/assets/1318111/26682533/ab4e100c-46d8-11e7-8bb6-06d4c9995bcd.png)